sql/001_users.sql

```diff
@@ -1,4 +1,4 @@
 GRANT USAGE ON *.* TO 'ht_repository'@'%' IDENTIFIED BY 'ht_repository';
 GRANT SELECT ON `ht_rights`.* TO 'ht_repository'@'%';
-GRANT SELECT, INSERT, UPDATE, DELETE, LOCK TABLES ON `ht_repository`.* TO 'ht_repository'@'%';
-GRANT SELECT, INSERT, UPDATE, DELETE, LOCK TABLES ON `ht`.* TO 'ht_repository'@'%';
+GRANT ALL ON `ht_repository`.* TO 'ht_repository'@'%';
```

Comment:
This probably isn't necessary -- I had changed this because at one point it was doing a `TRUNCATE` in the tests, which needed drop table privilege, but now it's doing it with transactions.